UnicornRender/video/include/unicorn/video/Renderer.hpp

```diff
@@ -68,30 +69,29 @@ class Renderer
      */
     UNICORN_EXPORT virtual void SetDepthTest(bool enabled) = 0;
 
-    /** @brief  Creates new geometry mesh
-    *
-    *  Creates and subscribes to mesh.
-    *  Mesh shall be deleted via DeleteMesh().
-    *
-    *  @param [in] material describes mesh visual representation
-    *
-    *  @attention  Mesh lifetime is bound by its renderer's lifetime.
-    *              Using meshes after their renderer was destroyed is undefined behaviour.
-    *              If you're storing mesh pointers, consider storing them with a reference
-    *              to their renderer and listening for its Destroyed event for proper cleanup.
-    *
-    *  @return pointer to newly created geometry::Mesh
-    *
-    *  @sa DeleteMesh
+    /**
+    * @brief Adds mesh to rendering system
+    * @param [in] mesh mesh data
+    */
+    UNICORN_EXPORT virtual void AddMesh(Mesh* mesh) = 0;
```

Comment:
What happens if Renderer is destroyed and there are still meshes inside?

Reply:
Nothing, meshes are loaded / created and can be alive without renderer. If renderer destroyed, it destroys only internal data and meshes can be still usable if future.